lib/core/extension/proposal_model_extensions.dart

```diff
@@ -30,5 +30,6 @@ extension ProposalModelTimeFormatting on ProposalModel {
   double unityToPercent() => unity==null?0:unity!*.01;
   double commitmentToPercent() => commitment == null ? 0 : commitment! * .01;
   bool isExpired() => expiration!.toLocal().isBefore(DateTime.now());
+  bool isPassing()=>quorumToPercent()>=.2 && unityToPercent()>=.8;
```

Comment:
the values .2 and .8 are still hard-coded here?

Reply:
Yes, the values .2 and .8 are still hard-coded for now. I've added a TODO in the code to update these values later by fetching them dynamically from the server, as they are relative to each DAO.